flow/templates/stampede2.sh

```diff
@@ -20,6 +32,27 @@
 {% endblock %}
 
 {% block body %}
+{% if not use_mpi %}
+{% if parallel%}
+{{("Warning! Bundle submission without mpi on stampede2 is using launcher and --parallel is ignored")|print_warning}}
```

Comment:
Could you use the following formulation?

> Bundled submission without MPI on Stampede2 is using launcher; the `--parallel` option is therefore ignored.

Please note that I intentionally removed the leading "Warning!".